### ISSUE TITLE
Add error notice for switching through nodes when node is connected to multiple ways

### ISSUE THREAD

**Issue by danieldegroot2** [openstreetmap/iD#9818]
### Description

When using the follow mode ( `F` ), a warning is given to the user when nodes are part of multiple ways.

Such a warning is not given when attempting to switch through nodes ( `[` / `]` ).
Giving such a warning could improve mapper feedback. It is a very minor improvement though.

**Comment by tyrasd** [comment]
Yeah, such a message would be useful when starting at an intersection vertex to cycle through the nodes with `[` / `]`.

**Comment by ThorOdinson246** [comment]
Hi, @danieldegroot2 @tyrasd could you please provide more details? I am having trouble reproducing the issue. A screenshot or an example location/walkthrough would be very helpful to get started on this.

**Comment by maishivamhoo123** [comment]
#@team can you please assign this issue to me?

**Comment by Razen04** [comment]
I can work on this if no one is working on it.

**Comment by Razen04** [comment]
I plan to flash a suggestion when a user reaches a node in the middle of a path that belongs to multiple ways to either switch the ways manually or continue on the existing way.

At first/last nodes, no info/warning will appear. Does this match the intended behavior, or should the last node show a info like this is the last node in this diection in this way?

**Comment by Razen04** [comment]
This is how it looks as of now, if this is fine then I can create a PR.

https://github.com/user-attachments/assets/7b8c8c36-6548-41dd-ac98-84736f33100c
